docs/_documentations/mdt-eclipse-importedprojects.md

```diff
@@ -0,0 +1,192 @@
+
+For example:
+```sh
+FROM ibmjava:8-sfj
+    MAINTAINER IBM Java engineering at IBM Cloud
+
+    COPY target/spring-application-2.0.0.BUILD-SNAPSHOT.jar /app.jar
+
+    ENV JAVA_OPTS=""
+    ENTRYPOINT [ "sh", "-c", "java $JAVA_OPTS -Djava.security.egd=file:/dev/./urandom -jar /app.jar" ]
+```
+
+## Node.js projects
+
+Your `package.json` must meet the following requirements. For an example of a good `package.json` for importing, see [the express application generator](https://github.com/ibm-developer/generator-ibm-core-node-express/blob/develop/app/templates/package.json):
+
+- Ensure the project provides a `start` npm script in `package.json` so it can be started by `npm start`.
+    - For example: `start: "node server/server.js"`
+- In local installations of Codewind, you can restart the project in Debug mode. To use this feature, the project must also provide a `debug` npm script, which accepts connections from all hosts on port 9229. For help configuring this script, see [the Node.js debugging guide](https://nodejs.org/en/docs/guides/debugging-getting-started/#command-line-options).
+    - For example: `debug: "node --inspect=0.0.0.0:9229 server/server.js"`
+- If auto-build is enabled, `nodemon` is used to restart your project automatically. `nodemon` calls either the `start` or `debug` script on a code change, depending on whether or not the project is debugging. Consequently, neither of these scripts should invoke `nodemon`.
+    - If a problem occurs with either script, the error appears in the Application Logs view in Codewind.
+
+If you have a `Dockerfile`, it must meet the following requirements:
+- A `Dockerfile` is generated if it does not exist. Ensure the `Dockerfile` exposes your application port.
+    - For example, if you're using port 3000, the `Dockerfile` needs to include `EXPOSE 3000`.
+- Ensure that the application is located in the `/app` directory within the Docker container.
+- Ensure that the `Dockerfile` sets the working directory to `/app`:
+    - `WORKDIR "/app"`
+
+
+## Swift projects
+
+Codewind works with Swift projects that use the Kitura web framework.
+
+Requirements:
+
+- A `Dockerfile-tools` file is generated to build the project. Ensure the project can be built by using a `release` build configuration.
+For example, you should be able to build the project by using the command
+`swift build --configuration release`.
+- A `Dockerfile` file is generated. It runs the application that was built by using `Dockerfile-tools`.
+
+## Generic Docker projects
+
+If you have a Dockerized application that doesn't fit an existing template, you can still import the project into Codewind by selecting the **Docker** import option. For the application state detection to work, the Dockerfile needs to include an `EXPOSE` instruction to point to the port that is used to determine whether the project is running.
```

Comment:
Change `..by selecting the **Docker** import option` to `..by selecting the **Other (Basic Container)** as the project type`

Reply:
Changed